specs/src/interoperability/ethereum-bridge.md

```diff
@@ -30,363 +42,8 @@ Ethereum side.
-
-## Namada Bridge Relayers
-
-Validator changes must be turned into a message that can be communicated to
-smart contracts on Ethereum. These smart contracts need this information
-to verify proofs of actions taken on Namada.
-
-Since this is protocol level information, it is not user prompted and thus
-should not be the responsibility of any user to submit such a transaction.
-However, any user may choose to submit this transaction anyway.
-
-This necessitates a Namada node whose job it is to submit these transactions on
-Ethereum at the conclusion of each Namada epoch. This node is called the
-__Designated Relayer__. In theory, since this message is publicly available on the blockchain,
-anyone can submit this transaction, but only the Designated Relayer will be
-directly compensated by Namada.
-
-All Namada validators will have an option to serve as bridge relayer and
-the Namada ledger will include a process that does the relaying. Since all
-Namada validators are running Ethereum full nodes, they can monitor
-that the message was relayed correctly by the Designated Relayer.
-
-During the `FinalizeBlock` call in the ledger, if the epoch changes, a
-flag should be set alerting the next block proposer that they are the
-Designated Relayer for this epoch. If their message gets accepted by the
-Ethereum state inclusion onto Namada, new NAM tokens will be minted to reward
-them. The reward amount shall be a protocol parameter that can be changed
-via governance. It should be high enough to cover necessary gas fees.
-
-## Ethereum Smart Contracts
-The set of Ethereum contracts should perform the following functions:
-- Verify bridge header proofs from Namada so that Namada messages can
-  be submitted to the contract.
-- Verify and maintain evolving validator sets with corresponding stake
-  and public keys.
-- Emit log messages readable by Namada
-- Handle ICS20-style token transfer messages appropriately with escrow &
-  unescrow on the Ethereum side
-- Allow for message batching
-
-Furthermore, the Ethereum contracts will whitelist ETH and tokens that
-flow across the bridge as well as ensure limits on transfer volume per epoch.
-
-An Ethereum smart contract should perform the following steps to verify
-a proof from Namada:
-1. Check the epoch included in the proof.
-2. Look up the validator set corresponding to said epoch.
-3. Verify that the signatures included amount to at least 2 / 3 of the
-   total stake.
-4. Check the validity of each signature.
-
-If all the above verifications succeed, the contract may affect the
-appropriate state change, emit logs, etc.
-
-## Starting the bridge
-
-Before the bridge can start running, some storage may need to be initialized in 
-Namada. 
-
 ## Resources which may be helpful:
 - [Gravity Bridge Solidity contracts](https://github.com/Gravity-Bridge/Gravity-Bridge/tree/main/solidity)
-- [ICS20](https://github.com/cosmos/ibc/tree/master/spec/app/ics-020-fungible-token-transfer)
+- [ICS20]
```

Comment:
Add link?

Reply:
there's already a link on the file, it's just defined elsewhere